src/cascadia/TerminalSettingsModel/defaults.json

```diff
@@ -637,6 +638,7 @@
         { "keys":"ctrl+shift+p", "id": "Terminal.ToggleCommandPalette" },
         { "keys":"win+sc(41)", "id": "Terminal.QuakeMode" },
         { "keys": "alt+space", "id": "Terminal.OpenSystemMenu" },
+        { "keys": "ctrl+period", "id": "Terminal.QuickFix" },

```

Comment:
> The `quickFix` action is bound to `ctrl+period` by default to align with VS' "quick actions" feature and VS Code's "quick fix" feature.

@zadjii-msft funny enough, I feel like `openSuggestions` would be better suited here. Especially since the suggestions UI loads quick fixes automatically. Maybe we want ctrl+period to be used for openSuggestions instead?

Also, we're in a weird spot rn. Quick fixes are behind a feature flag so idk if it makes sense to bind this just yet. Happy to remove changes to this file, just want some clarity on the plan here.

CC @DHowett

Reply:
(In case you're curious) Found that "ctrl+period" is used for "repeat the last command" in nano, so yeah, we need a different keybinding.

Changed to "ctrl+shift+period".

The questions above still stand though. How about long-term we do...
- ctrl+shift+period --> quick fix
- alt+shift+period --> suggestions
Just wanna make sure I don't claim a keybinding you were planning to claim later.

Reply:
Discussed a bit during today's bug bash. Here's the leading proposal rn:
- default kbd opens suggestions UI (source=all)
- clicking QF button --> open suggestions UI

As we add more quick fixes (i.e. actual winget suggestions, non-snippets, etc.) and non-snippet suggestions, we can revisit this and change it.